backend-shared/config/db/db_field_constants.go

```diff
@@ -142,6 +142,7 @@ var DbFieldMap = map[string]int{
 	"DeploymentToApplicationMappingNamespaceLength":                           DeploymentToApplicationMappingNamespaceLength,
 	"DeploymentToApplicationMappingDeploymentNamespaceLength":                 DeploymentToApplicationMappingNamespaceLength,
 	"DeploymentToApplicationMappingWorkspaceUIDLength":                        DeploymentToApplicationMappingWorkspaceUIDLength,
+	"DeploymentToApplicationMappingNamespaceUIDLength":                        DeploymentToApplicationMappingWorkspaceUIDLength,
```

Comment:
Is this not duplicated with other changes in place? The fix for this is already in #108